packages/docs/package.json

```diff
@@ -4,6 +4,12 @@
   "description": "API documentation in JSON form.",
   "main": "dist/index",
   "types": "dist/index",
+  "exports": {
+    ".": "./dist/index.js",
+    "./functions": "./dist/functions/index.js",
+    "./types": "./dist/types/index.js",
+    "./matchers": "./dist/matchers/index.js"
+  },
```

Comment:
The annotations requires this because it categorizes the docs by importing the individual folders instead of getting them through the main entrypoint. Also, it seems to require file extensions and only works with the `NodeNext` module resolution.